### ISSUE TITLE
# add compatibility to MacOS

### ISSUE THREAD

**Issue by 111andre111** [apangin/jattach#21]
At the moment jattach apparently is compatible to Windows, Linux and FreeBSD.
I would like to add functionality to MacOS, so that it is possible to use it under Mac as well.

**Comment by apangin** [comment]
Thank you for your interest in jattach.

I see no problem compiling jattach as is on macOS. `make` seems to work fine on a newly set up system with Xcode installed. Tested on all versions from OS X 10.11 El Capitan to macOS 10.14 Mojave.

Therefore I'm not sure I understand the purpose of the given pull request. Setting CXX flags does not make much sense to me, as jattach is written in C, not C++.

**Comment by 111andre111** [comment]
Hi,

actually I found that I mixed something.
You know, under that link here:
https://github.com/apangin/jattach/releases/tag/v1.5
I didn't find only one executable for linux.
```
$ ./jattach
exec format error: ./jattach
 $ file ./jattach
./jattach: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.6.4, with debug_info, not stripped
```
But I don't know why, but back then that I tried to compile under Mojave it didn't compile, but got an error and after adding the snippet it was working, apart that maybe it was a bit too many flags.
Now it's working without flag...
This is super interesting for me and I don't know why this happened and I don't have this error anymore as well. Hm, then I think I will cancel the PR and close this ticket, apart from, that maybe it could be cool if you offer a compiled release for MacOS as well?
